### DISCUSSION TITLE
Release both amd64 and x86_64 archives (for back-compatibility)

### DISCUSSION THREAD

**Discussion by i4ki** [goreleaser/goreleaser#4171] [Q&A] [answered]
Hey,
I have a weird use case and I hope there's a workaround to make it work with goreleaser.
I have been using goreleaser for quite a while and we were renaming our archives for `amd64` and `386` into `x86_64` and `i386`, respectively. The problem is that our tool is mostly used together with Terraform and other iac tools which release their binaries using `amd64` and `386` in the archive name...
so our users are asking us to follow what other tools are doing so their CI/CD pipelines are easier, ie not having to setup different env vars in different places for installing similar tools. But our tools are already been installed by homebrew and asdf (and maybe others) package managers which already rely on x86_64 and i386, also many CI/CD out there already depend on those arch names.
So we want to distribute both arch styles for back-compatibility purposes. Is that possible?

In other words, is there a way of distributing both `amd64` and `x86_64` (duplicating the assets)? Same for `386` and `i386`?

I read the docs multiple times and tried some hacks, but none seems to work.

**Comment by caarlos0** [comment, marked as answer]
hey, the `archives` section is a list, you can add more than one entry there, each with its own name template scheming. I believe that it should work.

Just be careful to not create multiple archives with the same name.
